database/factories/BookingFactory.php

```diff
@@ -11,40 +11,27 @@
 
 class BookingFactory extends Factory
 {
-    /**
-     * The name of the factory's corresponding model.
-     *
-     * @var string
-     */
     protected $model = Booking::class;
 
-    /**
-     * Define the model's default state.
-     *
-     * @return array
-     */
     public function definition()
     {
         return [
             'slot_number'           => $this->faker->numberBetween(0, 10),
+            'user_id'               => randomOrCreate(app('user')),
+            'order_id'              => randomOrCreate(app('order')),
             'total_participants'    => $this->faker->numberBetween(0, 10),
             'total_amount'          => $this->faker->numberBetween(0, 10),
             'amount'                => $this->faker->numberBetween(0, 10),
             'total_taxes'           => $this->faker->numberBetween(0, 10),
             'total_fees'            => $this->faker->numberBetween(0, 10),
             'booking_category_id'   => randomOrCreate(app('booking_category')),
-            'rate_id'               => randomOrCreate(app('booking_category')), //TODO change class
-            'experience_id'         => $this->faker->numberBetween(0, 10),
-            'experience_type'       => 'Game',
-            'order_id'              => $this->faker->numberBetween(0, 10),
-            'order_type'            => 'Order',
-            'booking_status_id'     => randomOrCreate(app('status')),
-            'agent_id'              => $this->faker->numberBetween(0, 10),
-            'agent_type'            => 'Agent',
-            'user_id'               => $this->faker->numberBetween(0, 10),
-            'user_type'             => 'User',
-            'subject_id'            => $this->faker->numberBetween(0, 10),
-            'subject_type'          => 'Subject',        
+            'rate_id'               => randomOrCreate(app('rate')),
+            'experience_id'         => randomOrCreate(app('escaperoom_slot')),
```

Comment:
@wolfrednicolas we need `Experience` that implements `BookingExperienceInterface`.